tests/runtime-tests/src/services/python.rs

```diff
@@ -0,0 +1,65 @@
+use super::Service;
+use anyhow::Context as _;
+use std::{
+    collections::HashMap,
+    path::Path,
+    process::{Command, Stdio},
+};
+
+pub struct PythonService {
```

Comment:
Not about the code but more about the preference for Python - it's relatively hard to write cross platform Python and also have reliable dependency management that will be consistently available - we probably need some guidelines to go along with this (I may have missed some?)

Reply:
I think the issue is that we don't have a great alternative. Ideally whatever we pick has these properties:
* Relatively widely known or easy to write by cargo culting
  * e.g., Lua might tick some of these other boxes but is a bit of a strange pick
* Can start running relatively quickly 
  * this is why I didn't chose Rust despite the rest of the project being written in it
* Is possible to run cross platform
  * e.g., I'd like to avoid bash scripts so that we can run this easily on Windows
 * Can be used consistently
   *  this is why I didn't pursue using WebAssembly for this since I'd rather not have each service written in a different language